package.json

```diff
@@ -78,6 +79,7 @@
     "easy-pie-chart": "^2.1.7",
     "gaugeJS": "^1.3.7",
     "md5": "^2.3.0",
-    "throttle-debounce": "^3.0.1"
+    "throttle-debounce": "^3.0.1",
+    "uplot": "^1.6.22"
```

Comment:
No need to add `uplot` to `devDependencies`, since we need it as a `dependency` and it is already added there! 😄